finch/src/services/email.js

```diff
@@ -0,0 +1,51 @@
+import nodemailer from 'nodemailer';
+
+import config from '../../config/index.js';
+
+const { emailSender, smtp } = config;
+
+export default class Email {
+
+  constructor() {
+    this.sender = emailSender;
+  }
+
+  init() {
```

Comment:
why shouldn't this be merged with the constructor?

Reply:
I'm also thinking the same now, can't think of a reason of why this shouldn't be part constructor